src/NUnitEngine/nunit.engine/Internal/ServerUtilities.cs

```diff
@@ -88,12 +95,13 @@ public static TcpChannel GetTcpChannel()
         /// </summary>
         /// <param name="name">The name of the channel</param>
         /// <param name="port">The port to use if the channel must be created</param>
+        /// <param name="currentMessageCounter">An optional counter to provide the ability to wait for all current messages</param>
         /// <returns>The specified <see cref="TcpChannel"/> or null if it cannot be found and created</returns>
-        public static TcpChannel GetTcpChannel( string name, int port )
```

Comment:
Extra spaces in various method calls go against the [Coding Standard](https://github.com/nunit/docs/wiki/Coding-Standards#spaces)